File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int n = nums.length;
        if(n <= 1){
            return true;
        }
        int invCnt = 0;
        for(int i = 1; i< n; i++){
            if(nums[i] < nums[i-1]){
                invCnt += 1;
            }
        }
        if(nums[0] < nums[n-1]){
            invCnt += 1;
        }
        return invCnt <= 1;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n = len(nums)
        for rot_off in range(n):
            check_sorted = []
            for idx in range(rot_off, n):
                check_sorted.append(nums[idx])
            for idx in range(rot_off):
                check_sorted.append(nums[idx])
            is_sorted= True
            for i in range(1, n):
                if(check_sorted[i-1] > check_sorted[i]):
                    is_sorted = False
                    break

            if is_sorted: return True
        return False